File: be/src/main/java/com/netcracker/sd4alexanderrodko/sd4parent/entity/Student.java
package com.netcracker.sd4alexanderrodko.sd4parent.entity;

public class Student {
    private Account account;
    private long groupId;

    public Student() {
    }

    public Account getAccount() {
        return account;
    }

    public void setAccount(Account account) {
        this.account = account;
    }

    public long getGroupId() {
        return groupId;
    }

    public void setGroupId(long groupId) {
        this.groupId = groupId;
    }

}

File: frontend/src/app/model/student.ts
import {Account} from "./account";

export class Student {

  account: Account;
  groupId: number;

  constructor() {
    this.account = new Account();
  }


  static clone(student: Student): Student {
    let cloned: Student = new Student();
    cloned.account = student.account;
    cloned.groupId = student.groupId;
    return cloned;
  }

}
